modules/assim.batch/R/pda.utils.R

```diff
@@ -256,12 +256,23 @@ pda.load.priors <- function(settings, con, path.flag = TRUE) {
     for (i in seq_along(settings$pfts)) {
       
       files <- dbfile.check("Posterior", settings$pfts[[i]]$posteriorid, con, settings$host$name)
+      
       pid <- grep("post.distns.*Rdata", files$file_name)  ## is there a posterior file?
       if (length(pid) == 0) {
         pid <- grep("prior.distns.Rdata", files$file_name)  ## is there a prior file?
       }
       if (length(pid) > 0) {
         prior.paths[[i]] <- file.path(files$file_path[pid], files$file_name[pid])
+      } else {
```

Comment:
as I said in the gitter, if there are posterior objects in your workflow folders but the code can't find them, that indicates a problem that needs to be solved differently. PDA should be able to run as long as valid posterior/prior objects exist in a place where the code can load them, it shouldn't matter if they're produced in the same workflow. 

This looks like a hack to me. Now the `pid` obtained above and the object that's being loaded here doesn't have to be the same. PDA would run this way, but I'm not sure if this is what we want